packages/device_info/device_info/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.4.2+7
```

Comment:
I just noticed these two changes in the core device_info/device_info package, they're not needed for this PR, please remove them!

Reply:
OK this will be done.